tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -36,10 +37,50 @@ public function testFix($expected, $input = null, array $configuration = null)
         if (null !== $configuration) {
             $this->fixer->configure($configuration);
         }
+        $indent = '    ';
+        $lineEnding = "\n";
+        if (null !== $input) {
+            if (strpos((string) $input, "\t") !== false) {
```

Comment:
>@param null|string $input

so either the cast is overcomplete (`null !== $input` so must be s string), or the PHPDoc is of